### DISCUSSION TITLE
Using vcpkg as a thin wrapper around an msi?

### DISCUSSION THREAD

**Discussion by criddell** [microsoft/vcpkg#37978] [Q&A] [answered]
My project depends on the newest version of the Microsoft SQL Server ODBC driver.

Is there a way I can use vcpkg to wrap the msi installer so that other people who work on the project get the driver installed automatically the first time they build and get updates as they become available?

**Comment by Osyotr** [comment, marked as answer]
You can execute arbitrary commands in profile but keep in mind that touching system outside of vcpkg will likely require elevated rights.

**Reply by criddell** [reply]
I didn't think about the need for admin rights...

I'm starting to think vcpkg isn't the right solution. I'll probably just add a comment in the source where I #include the header file to let the user know if this doesn't compile, install msodbc.msi (or whatever it is).
